docs/src/Contributing.md

```diff
@@ -61,12 +61,23 @@ Create a branch for your feature; this will hold your contribution:
 $ git checkout -b <branchname>
 ```
 
-## Develop your feature
+#### Some useful tips
+- When you start working on a new feature branch, make sure you start from master by running: `git checkout master`.
+- When you create a new branch and check it out, as in `git checkout -b <branchname>`, a common convention is to make `branchname` something along the lines of `<loginname>/<affected-module>-<short-description>`.
```

Comment:
We've been trying to keep to within 80 columns for documentation source as well; could you adjust this and the additions below as well please?

Reply:
Oh, but I tried it and in a bullet point, if you go to a new line, it will exit the bullet point and will look like a new paragraph -- Awkward

Reply:
If you indent the second line to past the hyphen, it will format correctly.

Reply:
Oh, right! I forgot about it! Thanks for reminding me